model/src/wav_shr_mod.F90

```diff
@@ -1343,6 +1374,51 @@ subroutine ymd2date_long(year,month,day,date)
     if (year < 0) date = -date
   end subroutine ymd2date_long
 
+  integer function get_minimum_timestep(gcomp, rc)
+    ! Get the minimum timestep interval in seconds based on the nuopc.config variables *_cpl_dt,
+    ! if none of these variables are defined this routine will throw an error
+    type(ESMF_GridComp), intent(in) :: gcomp
+    integer, intent(out)            :: rc
+
+    character(len=CS) :: cvalue
+    integer                 :: comp_dt             ! coupling interval of component
+    integer, parameter :: ncomps = 8
+    character(len=3),parameter :: compname(ncomps) = (/"atm", "lnd", "ice", "ocn","glc","rof", "wav", "esp"/)
```

Comment:
Why is this routine in wav share code and not in the general share code? I thought there was actually a routine now in share that could deal with this.

Reply:
removed